opentracing-spring-rabbitmq-it/src/test/java/io/opentracing/contrib/spring/rabbitmq/RabbitWithoutRabbitTemplateConfig.java

```diff
@@ -133,6 +133,7 @@ private void sendReplyIfRequested(Message message) {
         if (addCustomResponseErrorMarkerHeader != null) {
           replyMessage.getMessageProperties().setHeader(HEADER_CUSTOM_RESPONSE_ERROR_MARKER_HEADER, "dummy error message");
         }
+        replyMessage.getMessageProperties().setCorrelationId(messageProperties.getCorrelationId());
```

Comment:
Required because the spring-rabbit-2.1.4 RabbitTemplate line 2515 now raises an exception if correlationId is not populated as opposed to previously where it would simply log an error.